testsuite/domain/src/test/java/org/jboss/as/test/integration/domain/AdminOnlyPolicyTestCase.java

```diff
@@ -68,6 +68,8 @@ public class AdminOnlyPolicyTestCase {
     private static final long initTime = System.currentTimeMillis();
     private static int slaveCount;
 
+    private static int exitCodeTimeout = TimeoutUtil.adjust(5);
```

Comment:
Is there any reason not to make this quite a bit higher, e.g. 20-30? If there's no particular requirement that the process exits quickly and if it doesn't that means there's a bug, then a higher timeout protects against odd environmental problems without leading to longer test runs or tests passing when they shouldn't.

Reply:
Hi @bstansberry , there is no specific reason, I just only thought that 5 seconds should be enough waiting for an exit code. I don't see any problems if we use a higher value, so I'm going to update it